package.json

```diff
@@ -19,7 +19,8 @@
     "docker:compose:test:down": "docker-compose -f docker-compose.yml -f docker-compose.test.yml down",
     "migrate:test": "NODE_ENV=test sequelize db:migrate --config src/config/databases.js --migrations-path src/db/migrations",
     "docker:workers:test": "docker exec -it resonate-api node src/jobs/queue-worker.js run",
-    "docker:seed:all:test": "docker exec -it resonate-api npx sequelize db:seed:all --config src/config/databases.js --seeders-path src/db/seeders/test",
+    "copy:testAudioFiles": "cp test/media/audio/* data/media/audio",
+    "docker:seed:all:test": "yarn copy:testAudioFiles && docker exec -it resonate-api npx sequelize db:seed:all --config src/config/databases.js --seeders-path src/db/seeders/test",
```

Comment:
does this succeed if the files already exist in `data/media/audio`? If it doesn't succeed does it fail all the next commands too?